external/jacoco/dockerfile/jacoco-test.sh

```diff
@@ -48,4 +48,6 @@ echo "Build jacoco completed"
 find ./ -type d -name 'surefire-reports' -exec cp -r "{}" /testResults \;
 echo "Test results copied"
 
-exit $test_exit_code
\ No newline at end of file
+exit $test_exit_code
+
```

Comment:
This file does not need changing, you can remove the extra lines that you have added at the end of it.